rllib/core/learner/learner_group.py

```diff
@@ -40,6 +40,14 @@ def _get_backend_config(learner_class: Type["Learner"]) -> str:
     return backend_config
 
 
+def _is_module_trainable(module_id: ModuleID, batch: MultiAgentBatch) -> bool:
```

Comment:
what information will users use from the batch to determine if a module is trainable? ATM we don't pass a MABatch with env infos in it to the learner, which is what I think it might use.

Reply:
We do pass it, what do you mean? It is how it was used in the old stack too. For example based on some logic on sample batch you want to decide whether this policy is trainable.

Reply:
kk